### DISCUSSION TITLE
Lucid integration with knex-postgis cookbook link is not working

### DISCUSSION THREAD

**Discussion by kaypee90** [adonisjs/core#4355] [General]
This link https://docs.adonisjs.com/cookbooks/using-knex-postgis-with-lucid#document returns 404

**Comment by RomainLanz** [comment]
Hey @kaypee90! 👋🏻 

Correct link is https://v5-docs.adonisjs.com/cookbooks/using-knex-postgis-with-lucid

**Reply by kaypee90** [reply]
@RomainLanz it will be good to redirect https://docs.adonisjs.com/cookbooks/using-knex-postgis-with-lucid#document to this new subdomain since blog posts online and Google search still reference the old one
